go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/golang/protobuf v1.3.2
 	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645
-	github.com/ligato/vpp-agent v2.1.1+incompatible
+	github.com/ligato/vpp-agent v2.3.0+incompatible
```

Comment:
Hmm, are we sure here?

Reply:
Good catch.  The networkservicemesh/sdk/vppagent is using v2.3.0 ... so the shift is coming from that, but as you pointed out in IM... the build machinery needs to be updated for that.